homeassistant/components/deconz/sensor.py

```diff
@@ -217,7 +221,7 @@ def close(self):
         self.sensor = None
 
     @callback
-    def async_update_callback(self):
+    def async_update_callback(self, ignore_update=False):
         """Sensor state updated."""
```

Comment:
Shouldn't you check here for `ignore_update` too ?

Reply:
No it is just tracker for battery sensors, if this would pan out a real battery sensor would be created, that would be good :)